frontend/front/src/components/SurveyComponent/SurveyComponent.js

```diff
@@ -31,6 +33,7 @@ const SurveyComponent = ({
   formDefault,
   surveyStructure,
   onDelete,
+  isErrorSurvey,
```

Comment:
This unused prop should be removed - `isSurveyError` (line 252) serves this purpose.

Reply:
I saw some examples using isError. This is a boolean, but I'll change it to error. Thanks!